wieldview/README.txt

```diff
@@ -21,3 +21,7 @@ Wield image transformation: To apply a simple transformation to the item in
 hand, add the group “wieldview_transform” to the item definition. The group
 rating equals one of the numbers used for the [transform texture modifier
 of the Lua API.
+
+Disabling the feature in-game: If you want to hide the wielded item
+you can add an INT metadata to the player called "show_wielded_item" and set
+it to 2 (any other value will show the wielded item again)
```

Comment:
```suggestion
it to 2 (any other value will show the wielded item again).
```

Reply:
Not resolved yet.

Reply:
missing punctuations, don't we have other problems? :smirk: